atc/resource/run_script.go

```diff
@@ -131,8 +131,11 @@ func (resource *resource) runScript(
 				return err
 			}
 		}
-
-		return json.Unmarshal(stdout.Bytes(), output)
+		err := json.Unmarshal(stdout.Bytes(), output)
+		if err != nil {
+			return fmt.Errorf("%s\n\nwhen parsing resource response:\n\n%s", err, stdout.String())
+		}
+		return err
```

Comment:
If err is not nil, then it returns at line 136, so should this line just return `nil`?

Reply:
Yes, you're right. The only possible value for err is nil at line 138. I agree with you, it would more clear directly to return nil